src/App.vue

```diff
@@ -1,28 +1,82 @@
 <template>
   <div id="app">
-    <img alt="Vue logo" src="./assets/logo.png">
-    <HelloWorld msg="Welcome to Your Vue.js App"/>
+    <div class="container">
+      <h1>Punk API</h1>
+      <h2>Brewdog's expansive back catalogue of beer</h2>
+
+      <section v-if="errored">
+        <p>We're sorry, we're not able to retrieve this information at the moment, please try back later</p>
+      </section>
+
+      <section v-else>
+        <div v-if="loading">Loading...</div>
+
+        <div class="beer-list" v-if="beers">
+          <div class="beer" v-for="beer in beers" :key="beer.id">
+            <h3>{{ beer.name }}</h3>
+
+            <p>{{ beer.description }}</p>
+          </div>
+        </div>
+      </section>
+    </div>
   </div>
 </template>
 
 <script>
-import HelloWorld from './components/HelloWorld.vue'
+import axios from 'axios';
 
 export default {
   name: 'app',
-  components: {
-    HelloWorld
+  data() {
+    return {
+      beers: null,
+      loading: true,
+      errored: false,
+      errors: []
+    };
+  },
+  mounted() {
```

Comment:
It's better to make API calls in the `created` life cycle hook. This is the earliest you can fetch data. 
Use mounted if you need access to the DOM as this is when it's available.

FYI: https://alligator.io/vuejs/component-lifecycle/#mounting-dom-insertion